### ISSUE TITLE
Saved Button (Clicked)

### ISSUE THREAD

**Issue by Andrey-1992** [Andrey-1992/quick-news-ui#46]
As a software developer, I should be able to manage the saved button iteration to do not duplicate same saved news.
